src/tensorlake/function_executor/service.py

```diff
@@ -198,7 +199,7 @@ def initialize(
                 duration_sec=f"{time.monotonic() - start_time:.3f}",
                 # Don't log the exception to FE log as it contains customer data
             )
-            log_user_event_initialization_finished(event_details, success=False)
+            log_user_event_initialization_failed(event_details, error=e)
             # Print the exception to stderr so customer can see it there.
             traceback.print_exception(e)
```

Comment:
We're already printing the full backtrace here, do we need to remove this line and the comment above it?